recipes/flake8-junit-report/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "flake8-junit-report" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8890c1ebe0acb516fefacddec4b802bca9f89bb07db933e4ee3cd11ceaa1e8b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - flake8_junit = junit_conversor.__main__:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - flake8
+
+test:
+  requires:
+    - flake8
+    - nose
+    - scripttest
+    - tox
+  commands:
+    - nosetests
+
+outputs:
+  - name: {{ name|lower }}
+  - name: flake8_junit_report
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - {{ pin_subpackage( name|lower, exact=True) }}
```

Comment:
This assumes that the build string matches exactly. Looking at the logs it does but I know some recipes had issue and use `max_pin` instead.

The current recipe works but runs the `pip install` command twice.
It might be better to refactor it and do it like for example: https://github.com/conda-forge/setuptools_scm-feedstock/blob/main/recipe/meta.yaml
As far as I know, this is the more "standard" way to make a package available under 2 different names.

Reply:
No, this is the current standard (see my other recipe PRs in this repo for iterations with other maintainers.

Reply:
I found https://github.com/conda-forge/staged-recipes/pull/18964.
In that one, you use a `build_base.sh` script to avoid the `pip install` to run twice.
Or is there another one?

Reply:
Nah that one was bad, it's https://github.com/conda-forge/staged-recipes/pull/18963 we experimented at least 4-5 times to get to that